gemini/__init__.py

```diff
@@ -1,8 +0,0 @@
-import os
```

Comment:
there's a pure python version of `flatten` above this block, but in my testing the itertools version was faster. I think we could just import that one (it's python2.6+)